Demo/ViewController.swift

```diff
@@ -9,18 +9,45 @@
 import UIKit
 import BadgifySwift
 
-class ViewController: UIViewController {
+fileprivate let swiftColor = UIColor(red: 223/255, green: 92/255, blue: 67/255, alpha: 1)
 
+class ViewController: UIViewController {
+    
+    @IBOutlet private weak var badgedView: UIImageView!
+    
+    @IBOutlet weak var stepper: UIStepper!
+    
+    private let badgeView: BadgeView = {
+        let view = BadgeView()
+        view.badgeValue = 1
+        view.backgroundColor = .white
+        view.textColor = swiftColor
+        view.badgeFont = .systemFont(ofSize: 14)
+        view.minBadgeSize = CGSize(width: 25, height: 25)
+        view.xOffset = 5
+        view.yOffset = 5
+        return view
```

Comment:
hey for demo let's use the default values

Reply:
I think it's good to point out you can do that by using the properties. Your call though 😀